common/app/common/NewNavigation.scala

```diff
@@ -196,19 +209,19 @@ object NewNavigation {
 
     val uk = NavLinkLists(
       List(lifestyle, fashion, food, recipes, travel, loveAndSex, family),
-      List(home, health, women, tech)
```

Comment:
Can you remove the reference in https://github.com/guardian/frontend/blob/master/common/app/common/navlinks.scala#L105 as well